src/login/logind-button.c

```diff
@@ -12,10 +12,28 @@
 #include "fd-util.h"
 #include "logind-button.h"
 #include "logind-dbus.h"
+#include "logind-seat-dbus.h"
 #include "missing_input.h"
 #include "string-util.h"
 
-#define CONST_MAX5(a, b, c, d, e) CONST_MAX(CONST_MAX(a, b), CONST_MAX(CONST_MAX(c, d), e))
+#define KEY_MAX_INTERESTED KEY_RESTART
+
+static const int keys_interested[] = {
+        KEY_ESC,
+        KEY_LEFTCTRL,
+        KEY_LEFTSHIFT,
+        KEY_RIGHTSHIFT,
+        KEY_LEFTALT,
+        KEY_RIGHTCTRL,
+        KEY_RIGHTALT,
+        KEY_POWER,
+        KEY_LEFTMETA,
+        KEY_RIGHTMETA,
+        KEY_SLEEP,
+        KEY_SUSPEND,
+        KEY_POWER2,
+        KEY_RESTART
```

Comment:
and add a comment here that if somebody adds something here, they should possibly update _KEY_MAX_INTERESTED above too.

Reply:
also, please add a final `,` to the list of values, so that additions results in minimal patches

(we do that across our codebase)